{"name": "test_login_noExistUser_wrongPasswd[data1]", "status": "passed", "attachments": [{"name": "log", "source": "24fca5cf-32a5-42f5-b68d-a0cd5815eeb5-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'用户名': 'dsadsadsa', '密码': 'pytho', '预期': '请输入至少6位数的密码'}"}], "start": 1587021868657, "stop": 1587021869537, "uuid": "352347c5-08c0-4dbb-ae4c-50a370c621db", "historyId": "634fa2f876868d6c8f81ec03e35c966f", "testCaseId": "d35b8259a99be3473adfcaff1df8682d", "fullName": "TestCases.test_00_login.TestLogin#test_login_noExistUser_wrongPasswd", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_00_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-O2A05I4"}, {"name": "thread", "value": "18160-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_00_login"}]}